client/src/components/PacmanGame/gameCore.js

```diff
@@ -146,34 +146,33 @@ export const getRandomAdjacentAvailableCell = (gridState, currentLocation) => {
   return getRandomAdjacentAvailableCell(gridState, currentLocation);
 };
 
-const getGridwithWeights = (grid) => {
+export const getGridwithWeights = (grid) => {
   const gridwithWeights = grid.map((array) => {
     const newArray = array.map((element) => {
       let weight = element;
```

Comment:
Why assigning `element` to `weight` when not using it in future. Initialize with 0 and return 1 if entity a wall